eng/common/TestResources/New-TestResources-Wrapper.ps1

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Should we consider naming this something like `New-Test-Resources-Bootstrapper` to offer more context around how it's intended to be used?